test/Interactions/ES_C_test.py

```diff
@@ -0,0 +1,63 @@
+#
+# Copyright 2023 sitangshugk95@meen-flkmpr3.engr.tamu.edu
+#
+# ### MIT license
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+"""
+Created on Wed Feb 22 22:52:48 2023
+
+@author: sitangshugk95
+"""
+import numpy as np
+import matplotlib.pyplot as plt
+from sklearn.metrics import mean_squared_error
```

Comment:
please do not rely on new dependency to compute a mean squared error !
numpy or scipy is enough for that

Reply:
> Please remove or explain all the additional example files. We can also setup a zoom meeting for that

They were added in error - all unwanted files have been deleted.